website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -6,44 +6,47 @@
   </tr>
 </table>
 
-
 # The HTML Server Service
 
-Let's build the first service for our application:
-the HTML server!
-If we were building our Todo app as a traditional monolith,
-this would be the only code base
-and perform all of the application's functionality:
+Let's build the first service for our application: 
+the HTML server! 
+If we were
+building our Todo app as a traditional monolith, 
+this would be the only code base 
+and perform all of the application's functionality: 
 receiving requests,
-storing todo items, user accounts, and API tokens in the database,
-configuring and using the search engine,
-tracking user sessions and permissions,
-rendering HTML for the browser, JSON for the REST API,
-sending out emails,
+storing todo items, user accounts, and API tokens in the database, 
```

Comment:
Sorry to bother you about this again. Your text editor is adding trailing spaces to all lines here. Please configure it to trim trailing spaces on save. There are ways to do that for all common editor types. Google is your friend here. :)

Reply:
No sweat. It looks like turning the formatter off altogether caused some issues.